cli/asc.js

```diff
@@ -303,7 +303,7 @@ exports.main = function main(argv, options, callback) {
     // Append entries
     if (asconfig.entries) {
       for (let entry of asconfig.entries) {
-        argv.push(optionsUtil.resolvePath(entry, asconfigDir));
+        argv.push(path.relative(baseDir, optionsUtil.resolvePath(entry, asconfigDir)));
```

Comment:
Why is this necessary when `asconfigDir` is already relative to `baseDir`?

Reply:
`resolvePath` returns an absolute path.  This was mainly to output a relative path when `--showConfig` is passed.

Reply:
It returns an absolute path only if `entry` is already absolute, where I guess keeping it that way makes sense for debugging purposes (so one knows about where the entry comes from by looking at the asconfigs), or node resolution is used, which is debatable. In the latter case, the place to change it would be inside of `resolvePath` where `require.resolve` is called, though.

Reply:
Oh, wait, for entries node resolution isn't used anyway, so the `relative` here seems redundant.

Reply:
It can be used to get the asconfigDir, which returns an absolute path.

Reply:
What do you mean?

Reply:
When the extends field the file could be found with node resolution, which returns an absolute path, which means `path.join(entry, asconfigDir)` would return an absolute path.  But I just really want to get this finished so if you don't like it then I can take it out.

Reply:
Thanks for clarifying. I'm not necessarily opposed to changing node resolution to relative, but if we do so I think we should do it in `resolvePath` where `require.resolve` is used (and the path returned as absolute), so we don't break explicitly specified absolute paths given as the value of `extends`. Like, if a user specifies something absolute explicitly, I'd prefer to not mangle it but keep it as-is.

Reply:
Changed it in #1406 accordingly.

Reply:
very true.

Reply:
Just looked at your changes to #1406, and I think you should roll it back and just keep everything as is.  The issue was that I wanted the path relative to `baseDir` in asc.js, but in resolvePath, `baseDir` is passed in as `asconfigDir.` So we could adapt that to always pass in top level `baseDir` or just forget about it all together.

Reply:
Ok, let's remove the `path.relative` here then.